HousingSearchListener/HousingSearchListener.cs

```diff
@@ -8,19 +13,16 @@ namespace HousingSearchListener
 {
     public class HousingSearchListener
```

Comment:
I had envisage this class deriving from `BaseFunction`, but how you've done it works too.